### DISCUSSION TITLE
How should this project be build in IIS

### DISCUSSION THREAD

**Discussion by Hayababuza** [petrsvihlik/WopiHost#166] [Q&A]
Hi,

I am relatively new to ASP.Net and the Cloud Storage Partner Programme and am trying to understand and create a Wopi Host and Handler.
I saw this project and wanted to test it and wanted to ask if both projects WopiHost and WopiHost.Web should be published in the same folder or if one of them should be published in a subfolder.

Do I have to take care of anything else? I have already adjusted the URLs. But somehow I can't get them to work.

Thanks in advance

**Comment by petrsvihlik** [comment]
Hi, WopiHost.Web is just a testing application allowing you to interact with WopiHost. WopiHost is what you need for OWA/OO/CSPP to work. If you decide to deploy both, they should be hosted separately.
